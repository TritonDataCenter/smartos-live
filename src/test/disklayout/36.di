SCSI	disk00	MFG	DISK	600127266816	no	no
SCSI	disk01	MFG	DISK	600127266816	no	no
SCSI	disk02	MFG	DISK	600127266816	no	no
SCSI	disk03	MFG	DISK	600127266816	no	no
SCSI	disk04	MFG	DISK	600127266816	no	no
SCSI	disk05	MFG	DISK	600127266816	no	no
SCSI	disk06	MFG	DISK	600127266816	no	no
SCSI	disk07	MFG	DISK	600127266816	no	no
SCSI	disk08	MFG	DISK	600127266816	no	no
SCSI	disk09	MFG	DISK	600127266816	no	no
SCSI	disk0a	MFG	DISK	600127266816	no	no
SCSI	disk0b	MFG	DISK	600127266816	no	no
SCSI	disk0c	MFG	DISK	600127266816	no	no
SCSI	disk0d	MFG	DISK	600127266816	no	no
SCSI	disk0e	MFG	DISK	600127266816	no	no
SCSI	disk0f	MFG	DISK	600127266816	no	no
SCSI	disk10	MFG	DISK	600127266816	no	no
SCSI	disk11	MFG	DISK	600127266816	no	no
SCSI	disk12	MFG	DISK	600127266816	no	no
SCSI	disk13	MFG	DISK	600127266816	no	no
SCSI	disk14	MFG	DISK	600127266816	no	no
SCSI	disk15	MFG	DISK	600127266816	no	no
SCSI	disk16	MFG	DISK	600127266816	no	no
SCSI	disk17	MFG	DISK	600127266816	no	no
SCSI	disk18	MFG	DISK	600127266816	no	no
SCSI	disk19	MFG	DISK	600127266816	no	no
SCSI	disk1a	MFG	DISK	600127266816	no	no
SCSI	disk1b	MFG	DISK	600127266816	no	no
SCSI	disk1c	MFG	DISK	600127266816	no	no
SCSI	disk1d	MFG	DISK	600127266816	no	no
SCSI	disk1e	MFG	DISK	600127266816	no	no
SCSI	disk1f	MFG	DISK	600127266816	no	no
SCSI	disk20	MFG	DISK	600127266816	no	no
SCSI	disk21	MFG	DISK	600127266816	no	no
SCSI	disk22	MFG	DISK	600127266816	no	no
SCSI	disk23	MFG	DISK	600127266816	no	yes
USB	key0	USBMFG	USB Key	15737028608	yes	no
