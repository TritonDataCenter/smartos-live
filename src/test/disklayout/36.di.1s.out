fatal error: no acceptable raidz layout is possible with 35 disks and 1 spare
